File: codeforces/2026/236A-boy-or-girl.py

def main():
    username = input()

    # number of distinct characters is odd = boy; otherwise, girl
    if len(set(username)) % 2 == 0:
        print("CHAT WITH HER!")
    else:
        print("IGNORE HIM!")


if __name__ == "__main__":
    main()

File: codeforces/2026/236A-boy-or-girl.cpp
// https://codeforces.com/problemset/problem/236/A

#include <bits/stdc++.h>
using namespace std;

int main() {
    ios::sync_with_stdio(false);
    cin.tie(nullptr);

    string username;
    cin >> username;

    set<char> unique_chars;
    for (unsigned int i = 0; i < username.size(); i++) {
        unique_chars.insert(username[i]);
    }

    if (unique_chars.size() % 2 == 0) {
        cout << "CHAT WITH HER!";
    } else {
        cout << "IGNORE HIM!";
    }

    cout << "\n";
    return 0;
}

